earth_enterprise/src/fusion/tools/gee_test/publish_historical.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash -eu
+#
+# Copyright 2017 Open GEE Authors
```

Comment:
I think we've settled on "Open GEE Contributors" for this.  @tst-ccamp correct me if I'm wrong.